test/parallel/test-cluster-dgram-1.js

```diff
@@ -28,25 +28,37 @@ const assert = require('assert');
 const cluster = require('cluster');
 const dgram = require('dgram');
 
+function getPort(cb) {
+  const s = dgram.createSocket({ type: 'udp4', reuseAddr: true });
+  s.bind(0, () => {
+    const port = s.address().port;
+    s.close();
+    cb(null, port);
+  });
+}
 
 if (common.isWindows) {
   common.skip('dgram clustering is currently not supported ' +
-              'on windows.');
+    'on windows.');
```

Comment:
No worries. Thanks for taking the time to do this PR, and I hope the duplicate PR thing won't discourage you from doing more!